lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -37,7 +37,9 @@ class DOM {
       element.className = className;
     }
     Object.keys(attrs).forEach(key => {
-      element.setAttribute(key, attrs[key]);
+      if (attrs[key] !== undefined) {
```

Comment:
need to update the `attrs` type signature if adding this, but it would probably be better to keep it strict and make the caller correct it if they don't want an attribute